lib/bloc/withdraw/withdraw_funds_state.dart

```diff
@@ -2,24 +2,31 @@ import 'package:breez_translations/generated/breez_translations.dart';
 
 class WithdrawFundsState {
   final List<FeeOption> feeOptions;
-  final String errorMessage;
+  final String? reverseSwapErrorMessage;
```

Comment:
It looks like we are using this model for both reverse swap and withdraw funds. 
I think each one of them should have different state and different bloc, they are not the same thing.

Reply:
I had separated ReverseSwap & Sweep blocs first and then merged them later to use a singular bloc as they share fetching recommended fee & generating fee options logic but I agree the error message parts on the state are a bit confusing.

I'll separate them to avoid confusion and move the fee option logic to it's own bloc.